locales/hi.json

```
@@ -1,1542 +1,1542 @@
+  "Major ninth": "प्रमुख नवम",
+  "Diminished tenth": "घटित दशम",
+  "Minor tenth": "लघु दशम",
+  "Augmented ninth": "वृद्ध नवम",
+  "Major tenth": "प्रमुख दशम",
+  "Diminished eleventh": "घटित एकादश",
+  "Perfect eleventh": "पूर्ण एकादश",
+  "Augmented tenth": "वृद्ध दशम",
+  "Diminished twelfth": "घटित द्वादश",
+  "Augmented eleventh": "वृद्ध एकादश",
+  "Perfect twelfth": "पूर्ण द्वादश",
+  "Diminished thirteenth": "घटित त्रयोदश",
+  "Minor thirteenth": "लघु त्रयोदश",
+  "Augmented fifth, plus an octave": "वृद्ध पंचम, साथ में एक सप्तक",
+  "Major thirteenth": "प्रमुख त्रयोदश",
+  "Diminished seventh, plus an octave": "घटित सप्तम, साथ में एक सप्तक",
+  "1st 2nd 3rd 4th 5th 6th 7th 8th 9th 10th 11th 12th": "1वा 2वा 3वा 4वा 5वा 6वा 7वा 8वा 9वा 10वा 11वा 12वा",
+  "augmented": "बढ़ाया हुआ",
+  "diminished": "कम",
+  "perfect": "पूर्ण",
+  "chromatic": "क्रोमैटिक",
+  "algerian": "अल्जीरियन",
+  "spanish": "स्पेनिश",
+  "octatonic": "ऑक्टाटॉनिक",
+  "harmonic major": "हार्मोनिक मेजर",
+  "natural minor": "नैचुरल माइनर",
+  "harmonic minor": "हार्मोनिक माइनर",
+  "melodic minor": "मेलोडिक माइनर",
+  "dorian": "डोरियन",
+  "phrygian": "फ्रिजियन",
+  "lydian": "लिडियन",
+  "mixolydian": "मिक्सोलिडियन",
+  "locrian": "लोक्रियन",
+  "jazz minor": "जैज माइनर",
+  "bebop": "बीबॉप",
+  "arabic": "अरबी",
+  "byzantine": "बाइज़ंटाइन",
+  "enigmatic": "एनिगमैटिक",
+  "ethiopian": "इथियोपियन",
+  "geez": "गीज़",
+  "hindu": "हिन्दू",
+  "hungarian": "हंगेरियन",
+  "romanian minor": "रोमानियन माइनर",
+  "spanish gypsy": "स्पैनिश जिप्सी",
+  "maqam": "मकाम",
+  "minor blues": "माइनर ब्लूज",
+  "major blues": "मेजर ब्लूज",
+  "whole tone": "होल टोन",
+  "minor pentatonic": "माइनर पेंटाटॉनिक",
+  "major pentatonic": "मेजर पेंटाटॉनिक",
+  "chinese": "चाइनीज़",
+  "egyptian": "इज़िप्शियन",
+  "hirajoshi": "हिराजोशी",
+  "Japan": "जापान",
+  "in": "इन",
+  "minyo": "मिन्यो",
+  "fibonacci": "फिबोनाची",
+  "custom": "अनुकूलित",
+  "highpass": "हाईपास",
+  "lowpass": "लोपास",
+  "bandpass": "बैंडपास",
+  "highshelf": "हाईशेल्फ",
+  "lowshelf": "लोशेल्फ",
+  "notch": "नॉच",
+  "allpass": "ऑलपास",
+  "peaking": "पीकिंग",
+  "sine": "साइन",
+  "square": "वर्ग",
+  "triangle": "त्रिकोण",
+  "sawtooth": "सॉटूथ",
+  "even": "सम",
+  "odd": "विषम",
+  "scalar": "स्कैलर",
+  "piano": "पियानो",
+  "violin": "वायलिन",
+  "viola": "वायोला",
+  "xylophone": "क्सीलोफ़ोन",
+  "vibraphone": "वाइब्राफ़ोन",
+  "cello": "चेलो",
+  "double bass": "डबल बास",
+  "guitar": "गिटार",
+  "sitar": "सितार",
+  "harmoiunm": "harmoiunm",
+  "acoustic guitar": "एकॉस्टिक गिटार",
+  "flute": "बांसुरी",
+  "clarinet": "क्लैरिनेट",
+  "saxophone": "सैक्सोफोन",
+  "tuba": "ट्यूबा",
+  "trumpet": "ट्रंपेट",
+  "oboe": "ओबो",
+  "trombone": "ट्रॉम्बोन",
+  "electronic synth": "इलेक्ट्रॉनिक सिंथ",
+  "simple 1": "सिम्पल 1",
+  "simple 2": "सिम्पल 2",
+  "simple 3": "सिम्पल 3",
+  "simple 4": "सिम्पल 4",
+  "white noise": "व्हाइट नॉइज़",
+  "brown noise": "ब्राउन नॉइज़",
+  "pink noise": "पिंक नॉइज़",

```

can we actually translate these instead of transliterating?

Maybe that would be better 
We are assuming that the users of this translation won't know many English words other than the commonly used ones. 

```suggestion
  "white noise": "श्वेत नॉइज़",
  "brown noise": "भूरी नॉइज़",
  "pink noise": "गुलाबी नॉइज़",
```

also open to using सफ़ेद 
```suggestion
  "white noise": "सफ़ेद नॉइज़",
  "brown noise": "भूरी नॉइज़",
  "pink noise": "गुलाबी नॉइज़",
```